### PR TITLE
Restrict fossa

### PR DESCRIPTION
**Pull request by Dref360**
FOSSA cannot be run by external people, so we only use it on master.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -43,18 +39,12 @@ jobs:
             echo "export CI_BUILD_NUMBER=${CIRCLE_BUILD_NUM}" >> $BASH_ENV
             echo "export CI_NAME=circle" >> $BASH_ENV
 
-      - run:
-          name: FOSSA scan
-          command: fossa
-          working_directory: ~/baal
-
       # run tests!
```
Comment (orangetoaster) [inline .circleci/config.yml:42]: Tests are run in coverage now, update comment.

**Review by orangetoaster — DISMISSED** [review]
LGTM

File: .circleci/config.yml (hunk 1 above)
Comment (Dref360) [inline .circleci/config.yml:42]:
```suggestion
      # pep8
```

**Review by parmidaatg — APPROVED** [review]
LGTM

**Pull request merged**